lib/replicache.dart

```diff
@@ -276,6 +281,10 @@ class Replicache implements ReadTransaction {
 
     checkStatus(syncInfo['clientViewInfo'], 'client view');
 
+    if (reauth && getDataLayerAuth != null) {
```

Comment:
This results in sync getting delayed until the next poll interval. Why not try and sync again if the auth changes before returning?

Reply:
Good idea.